src/core/Util.js

```diff
@@ -104,6 +104,30 @@ module.exports = {
         toggleMarkdownModeBtn.style.display = 'block';
       }
     });
+  },
+
+  preventUploadedImagesDragging: function() {
+    var wysiwygDiv = document.querySelector(".wk-wysiwyg");
+    var self = this;
+
+    function handleChange() {
+      if (window.isScrollingDisabled) {
+        self.enableScroll();
+      }
+
+      var imageElements = document.querySelectorAll(
+        '.wk-wysiwyg img:not([draggable="false"])'
+      );
+
+      imageElements.forEach(imageElement => {
+        imageElement.setAttribute("draggable", "false");
+      });
```

Comment:
These 3 lines seem to cause 'ReferenceError: Can't find variable: PL'. I don't have any idea why that happens.

Reply:
Arrow function, right...